emulator/src/extensions/emu_args.c

```diff
@@ -89,6 +89,8 @@ emuArgs_t emulatorArgs = {
     .fuzzTouch   = false,
     .fuzzMotion  = false,
 
+    .headless = false,
```

Comment:
Did you mean to add `.playback` and `.record` here? Also `.recordFile` and `.replayFile`

Reply:
@dylwhich this is the only comment that matters

Reply:
Ah, I've been terrible about keeping these consistent, yeah.

Reply:
Wanna update before I merge or no?